src/tests/ftest/dfuse/container_attrs.py

```diff
@@ -0,0 +1,113 @@
+"""
+  (C) Copyright 2020-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from apricot import TestWithServers
+from dfuse_utils import get_dfuse, start_dfuse
+
+
+class DfuseContainerAttrs(TestWithServers):
+    """Check if the dfuse attributes of a container are properly managed.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # obtain separate logs
+        self.update_log_file_names()
+
+        # Start the servers and agents
+        super().setUp()
+
+    def _check_attrs(self, dfuse, container_name):
```

Comment:
We could pass the `TestContainer` object (as `container`) here instead of `container_name` with which we could replace:
```
        attrs = self.params.get("attrs", f"/run/{container_name}/*")
        for attr in [attr.split(':') for attr in attrs.split(",")]:
```
with:
```
        for attr in [attr.split(':') for attr in container.attrs.value.split(",")]:
```

Reply:
Also if this were to be used outside of this test we could consider adding it as a function to the TestContainer class itself.

Reply:
Because the regex `fr"^.+\ssetting\s+'{name}'\s+is\s+(\d+)\s+seconds$",` is specific to dfuse, I don't think this should be a helper method on the container. I think if we find some other test needs this, we can consider a helper method somewhere at that time.